loadtest/scripts/populate_genesis_accounts.py

```diff
@@ -34,45 +43,109 @@ def add_genesis_account(account_name, lock, local=False):
             "mnemonic": mnemonic,
         }
         json.dump(data, f)
-    success = False
-    retry_counter = 5
+
+    return add_account_cmd
+
+
+def create_genesis_account(account_index, account_name, local=False):
+    address, mnemonic = add_key(account_name=account_name, local=local)
+    add_account_cmd = add_account(account_name=account_name, address=address, mnemonic=mnemonic, local=local)
+
+    retry_counter = 0
     sleep_time = 1
-    while not success and retry_counter > 0:
+
+    while True and retry_counter < 1000:
         try:
-            with lock:
-                subprocess.check_call(
-                    [add_account_cmd],
-                    shell=True,
-                    timeout=20,
-                )
-                success = True
+            print(f'Running: ${add_account_cmd}')
+            subprocess.call(
+                [add_account_cmd],
+                shell=True,
+                timeout=20,
+            )
+            break
         except subprocess.CalledProcessError as e:
-            print(f"Encountered error {e}, retrying {retry_counter - 1} times")
-            retry_counter -= 1
+            print(f"Encountered error {e}, retried {retry_counter} times")
+            retry_counter += 1
             sleep_time += 0.5
             time.sleep(sleep_time)
+    
+    if retry_counter >= 1000:
+        exit(-1)
+    
+    global_accounts_mapping[account_index] = {
+        "balance": {
+            "address": address,
+            "coins": [
+                {
+                    "denom": "usei",
+                    "amount": "1000000000"
```

Comment:
fine to hard code for now but should update when this runs
https://github.com/sei-protocol/sei-chain/pull/320/files#diff-e1c1ef35431f7e66f33f0a100a17660ccb93ed9a36e479b660fd1931faf905adR33